ILMerge.Tests/TestFiles.cs

```diff
@@ -1,8 +1,29 @@
-﻿namespace ILMerging.Tests
+﻿using NUnit.Framework;
+using System.IO;
+
+namespace ILMerging.Tests
 {
     internal static class TestFiles
     {
-        public const string TestSnk = "test.snk";
-        public const string TestPfx = "test.pfx";
+        private static string FromCurrentDir(string fileName)
+        {
+            return Path.Combine(TestContext.CurrentContext.TestDirectory, fileName);
+        }
+
+        public static string TestSnk
+        {
+            get
+            {
+                return FromCurrentDir("test.snk");
+            }
+        }
```

Comment:
Feel free to ignore me if you want because I'm not the owner haha! I find it easier to read  
`public static string TestSnk => FromCurrentDir("test.snk");`
What's your opinion on that?